spec/helpers/application_helper_spec.rb

```diff
@@ -1628,4 +1628,31 @@
       end
     end
   end
+
+  describe '.view_to_table' do
```

Comment:
should this be `'.view_to_association'`?  Is it a class or instance method?  If instance method, then `describe '#view_to_association'`.